releasenotes/notes/use-remote-tagger-for-trace-and-security-agents-9650d1be5d49e47f.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    ``apm_config.remote_tagger``,  ``security_agent.remote_tagger``, and ``event_monitoring_config.remote_tagger`` config entries are deprecated and removed.
```

Comment:
Maybe we should be a bit more explicit that the config here is now that the remote_tagger is always in use (so setting this config to false has no effect)